pkg/option/config.go

```diff
@@ -711,6 +711,12 @@ const (
 	// IPSecKeyFileName is the name of the option for ipsec key file
 	IPSecKeyFileName = "ipsec-key-file"
 
+	// EnableIPSecEncrytpedOverlay is the name of the option which enables
+	// the EncryptedOverlay feature.
+	//
+	// This feature will encrypt overlay traffic before it leaves the cluster.
+	EnableIPSecEncrytpedOverlay = "enable-ipsec-encrypted-overlay"
```

Comment:
```suggestion
	EnableIPSecEncryptedOverlay = "enable-ipsec-encrypted-overlay"
```

Reply:
I knew there'd be at least one :laughing:

Reply:
fixed in latest push